test/clients.go

```diff
@@ -41,13 +41,18 @@ func NewSpoofingClient(ctx context.Context, client kubernetes.Interface, logf lo
 
 // BuildClientConfig builds the client config specified by the config path and the cluster name
 func BuildClientConfig(kubeConfigPath string, clusterName string) (*rest.Config, error) {
+	loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
 	overrides := clientcmd.ConfigOverrides{}
+
+	if kubeConfigPath != "" {
+		loadingRules.ExplicitPath = kubeConfigPath
+	}
```

Comment:
what's this changed for?

Reply:
This is what makes it so that if a user provides a kubeconfig via flags it will still get used